var/spack/repos/builtin/packages/py-ruamel-yaml-clib/package.py

```diff
@@ -14,6 +14,7 @@ class PyRuamelYamlClib(PythonPackage):
     homepage = "https://sourceforge.net/p/ruamel-yaml-clib/code/ci/default/tree/"
     pypi = "ruamel.yaml.clib/ruamel.yaml.clib-0.2.0.tar.gz"
 
+    version("0.2.7", sha256="1f08fd5a2bea9c4180db71678e850b995d2a5f4537be0e94557668cf0f5f9497")
     version("0.2.0", sha256="b66832ea8077d9b3f6e311c4a53d06273db5dc2db6e8a908550f3c14d67e718c")
 
     depends_on("python@2.7:2.8,3.5:", type=("build", "link", "run"))
```

Comment:
Could remove this line

Reply:
I was not sure because of the link dependency. While trying to see if it is needed or not I saw that the old release (py-ruamel-yaml@0.2.0) was not installing anymore (nothing to do with this PR) with the error:
```
  >> 1168    error: legacy-install-failure
```
But with `python@3.9` it works. I have changed the python dependencies now.